syllabus/5-Substrate/2-WASM-Meta-Protocol-Slides.md

```diff
@@ -53,7 +53,7 @@ Recall that the boundary for this division is the **state transition**
 
 ### Substrate: a short recap
 
-- **Host Functions**: Means of a runtime communicating with its host environment, i.e. the Substrate client.
+- **Host Functions**: Means of a runtime communicating with its host environment, i.e. the Substrate node.
```

Comment:
Should we call it node everywhere now? 

Personally I feel we should not try to avoid calling it any of the three (host/client/node) but explain to students why these terms are interchangeably used to describe a node. Eg: in context of a runtime, the node is a host to runtime. Though I agree if we have to settle on using one term throughout, node seems neutral.

Reply:
I changed all instances that I found to node, but indeed I clearly say that Node and Client and Host are equal and won't be super strict on using Node.

But for written content, it is very useful to have one standard.